unlock-protocol.com/src/pages/post.js

```diff
@@ -60,7 +60,8 @@ Post.propTypes = {
 }
 
 Post.getInitialProps = async context => {
-  return await preparePostProps(context)
+  const { slug } = context.query
+  return await preparePostProps(slug)
```

Comment:
I think the `context` object is specific to next so I'd rather not pass it down to `preparePostProps`

Reply:
:+1: